### ISSUE TITLE
Weird result wrt TLS_FALLBACK_SCSV

### ISSUE THREAD

**Issue by drwetter** [drwetter/testssl.sh#929]
Seems the check for TLS 1.2 only failed for some reason at L11494:

```
 SSLv2      not offered (OK)
 SSLv3      offered (NOT ok)
 TLS 1      offered
 TLS 1.1    not offered
 TLS 1.2    not offered
 TLS 1.3    not offered
```

and

```
 TLS_FALLBACK_SCSV (RFC 7507)              No fallback possible, TLS 1.2 is the only protocol (OK)
```

Not reproducible e.g. with Apache+OpenSSL (probably it is Tomcat+BC).

**Comment by drwetter** [comment]
should make use of ``has_server_protocol()``

**Comment by drwetter** [comment]
was addressed in #950
